src/components/inspector-tabs/editor.scss

```diff
@@ -14,3 +14,11 @@
 		height: 16px;
 	}
 }
+
+.stk-inspector-hint__google-map {
+	margin: 10px 0 10px !important;
+	span {
+		font-size: 12px;
+		padding: 6px !important;
+	}
+}
```

Comment:
I see some unnecessary styling here. Please do not change font sizes and other spacing unless necessary - for example, there is no need to change the padding to `6px` because our UI is using `16px` already, just add the missing padding. And the `font-size` is already provided by the UI, no need to change it. I suggest removing this specific styling for this google map hint and instead handle things for the hint element itself:

```suggestion
```

I see you're trying to add padding to the right because this doesn't have a close button, it's better to handle this generally: add this to the `span`

```css
.stk-inspector-hint {
	// Without a close button.
	span:last-child {
		padding: 16px;
	}
}
```

Since you're changing the margin because the default has a large top margin, then you can specify the margins to be different when placed beside panels vs when inside controls: adjust the css for hints to be

```css
.stk-inspector-hint {
	margin: 16px;
}
.components-panel__body + .stk-inspector-hint {
	margin-top: 32px;
}
```